addons/mail/static/src/discuss/core/common/channel_member_model.js

```diff
@@ -15,6 +15,12 @@ export class ChannelMember extends Record {
     create_date;
     /** @type {string} */
     custom_channel_name;
+    /**
+     * false means using the custom_notifications from user settings.
+     *
+     * @type {false|"all"|"mentions"|"no_notif"}
+     */
+    custom_notifications = false;
```

Comment:
We should not have default values in fields coming from the server, they should be undefined until we receive a value for them.